### ISSUE TITLE
TimeOut when running on Colab

### ISSUE THREAD

**Issue by jeromemassot** [philipperemy/Stanford-OpenIE-Python#31]
Hi Philippe,

I try to use the wrapper from Colab but I have a permanent issue when it tries to start the remote server.
PermanentlyFailedException: Timed out waiting for service to come alive.

I have installed the CoreNLP library with the English .jar and update the CLASSPATH, so if I can point to my local java module, it may work more easily than pointing to a remote server.

Is it possible to custom Stanford-OpenIE-Python wrapper to use a local install ?

Thanks

Best regards

Jerome

**Comment by philipperemy** [comment]
@jeromemassot I don't think you can run it on Colab (as it is). The library spins up a big java server (CoreNLP server) locally. I don't know how Colab works but chances are you can't do it because the ports are closed...
Related issue: https://github.com/philipperemy/Stanford-OpenIE-Python/issues/21.

One way to make it work would be to host the java server on a remote server (for example AWS, GCP) and give the IP:port to CoreNLPClient (argument `endpoint`). If you have time to try it, it could be great to see if it works.

https://github.com/stanfordnlp/python-stanford-corenlp/blob/4d0aa08521e13e2a1c707f5a7ddcdaa3ec530d86/corenlp/client.py#L125

**Comment by theunknownmonk** [comment]
Hi Phillipe!!
I tried to run the code given in readme in jupyter notebook after installing openie package.I got this error : FileNotFoundError: [WinError 2] The system cannot find the file specified. Can you please help me out?I am a newbie to python!! So please excuse me if I have made a very silly error!!
Thanks

**Comment by SaiAishu** [comment]
Hi, I have been running OpenIE from my local Jupyter notebook for the past month and it worked fine. I'm getting timeout errors today, was wondering if the server is down or something. If so, can you please provide a fix/workaround? Thanks.
<img width="745" alt="image" src="https://user-images.githubusercontent.com/30656747/186659284-c9ea8945-0b5e-4553-a4af-023751168f65.png">

PermanentlyFailedException: Timed out waiting for service to come alive.
